src/_pytest/terminal.py

```diff
@@ -647,9 +647,10 @@ def pytest_sessionfinish(self, exitstatus):
     def pytest_terminal_summary(self):
         self.summary_errors()
         self.summary_failures()
-        yield
         self.summary_warnings()
+        yield
```

Comment:
"short test summary info" comes via the skipping plugin, which I found rather confusing.
Should/can this be moved?

It would also might make sense to have "short test summary info" after "summary_passes" even, but I wanted to not touch that behavior (and am not using it - `-ra` does not include `P`).

Reply:
Hmmm if we move it to `pytest_terminal_summary`, will it simplify things? It doesn't look so simple to move it, because there's some logic in there to handle summaries for xpass, xfail, etc.

Reply:
Perhaps we should add a comment here explaining why we display warnings twice, what do you think?

Reply:
Makes sense.